### PR TITLE
Filters on search screen

### PR DESCRIPTION
**Pull request by mattwalo32**
## Summary
Move the department and university filtering from the home screen to the search screen. Did a bit of refactoring while I was at it.

### CODE REVIEW

File: src/containers/CTNavigationContainer/CTNavigationContainer.js [hunk 1]
```diff
@@ -136,27 +147,27 @@ const CTNavigationContainer = () => {
           }}
         />
         <Tab.Screen
-          name="Home"
+          name={STACK_SCREENS.STARRED}
           component={HomeNavigator}
           options={{
-            tabBarLabel: 'Home',
+            tabBarLabel: 'Starred',
```
Comment (Hangary) [inline src/containers/CTNavigationContainer/CTNavigationContainer.js:153]: Why not use `STACK_SCREENS.STARRED` here?

File: src/containers/CTNavigationContainer/CTNavigationContainer.js [hunk 2]
```diff
@@ -136,27 +147,27 @@ const CTNavigationContainer = () => {
...
           }}
         />
         <Tab.Screen
-          name="Starred"
-          component={StarredNavigator}
+          name={STACK_SCREENS.HISTORY}
+          component={HistoryNavigator}
           options={{
-            tabBarLabel: 'Starred courses',
+            tabBarLabel: 'History',
```
Comment (Hangary) [inline src/containers/CTNavigationContainer/CTNavigationContainer.js:163]: Why not use `STACK_SCREENS.HISTORY` here?

**Review by Hangary — APPROVED** [review]
Good job overall!

File: src/containers/CTNavigationContainer/CTNavigationContainer.js (hunk 1 above)
Comment (mattwalo32) [reply src/containers/CTNavigationContainer/CTNavigationContainer.js:153]: That's a good point, the only argument I have against that is that all of the `STACK_SCREENS` variable is meant to be used as an enum -- we use that in other parts of our code to specify the value of the screen that we want to push onto the navigator. So I think the underlying value of this enum shouldn't neccesarily be tied to the display name of the screen on the tab -- especially if translations were to ever be added into the app.

What are your thoughts on this?

**Review by Jagaskak — APPROVED** [review]
Looks good

**Pull request merged**